XMonad/Layout/IndependentScreens.hs

```diff
@@ -101,13 +112,27 @@ unmarshallW = snd . unmarshall
 workspaces' :: XConfig l -> [VirtualWorkspace]
 workspaces' = nub . map unmarshallW . workspaces
 
+-- | wsConvertToNum takes a workspace and returns its ScreenId
```

Comment:
`unmarshallS . W.tag` takes a workspace and returns its `ScreenId`.

Why do we need another function to do that?